Jenkinsfile

```diff
@@ -20,30 +20,35 @@ pipeline {
 
   stages {
     stage('Install') {
-      steps {
-        sh 'yarn install'
+      steps { 
+        script {
+          nix.develop('yarn install')
+        }
       }
     }
 
     stage('Build') {
       steps {
         script {
-          sh 'yarn build'
-          jenkins.genBuildMetaJSON('build/build.json')
+          nix.develop('yarn build')
+          jenkins.genBuildMetaJSON('dist/build.json')
```

Comment:
Why are we changing the output dir name?

Reply:
But wouldn't you have to modify the build setup to build into that folder and not `bulild`?

Reply:
Was this ever verified that the path is correct? Seems wrong to me.

Reply:
It is wrong. Changed to correct path.